cartography/data/jobs/cleanup/oci_import_policies_cleanup.json

```diff
@@ -0,0 +1,14 @@
+{
+  "statements": [{
+    "query": "MATCH (:OCITenancy{ocid: {OCI_TENANCY_ID}})-[:OCI_POLICY]->(n:OCIPolicy) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
During creation we assign OCIPolicy based on condition. 

MATCH (aa) WHERE (aa:OCITenancy OR aa:OCICompartment) AND aa.ocid={COMPARTMENT_ID}
    MERGE (aa)-[r:OCI_POLICY]->(pnode)

I don't see the OCICompartment case in the cleanup